scripts/examples/python/mini-xenrt.py

```diff
@@ -15,9 +15,9 @@
 
 class Operation:
     def __init__(self):
-        raise "this is supposed to be abstract, dummy"
+        raise Exception("this is supposed to be abstract, dummy")
```

Comment:
```suggestion
        raise NotImplementedError
````

Reply:
I will change this to reflect  NotimplementedError exception.

Reply:
Exception has been corrected ,commit id "862b8140d91f56e9a8e9fa4cceb83a319a5ca4d2"